spec/change_sets/recording_change_set_spec.rb

```diff
@@ -2,106 +2,31 @@
 require "rails_helper"
 
 RSpec.describe RecordingChangeSet do
-  subject(:change_set) { described_class.new(form_resource) }
```

Comment:
Why are there a bunch of tests removed? Are they duplicates or something? I wonder if this should be in a separate PR.

Reply:
Good catch - we had two recording_change_set specs, so my renaming killed one of them. They're combined now.